imsv-docs-astro/src/content/docs/guides/regions.mdoc

```diff
@@ -2,7 +2,18 @@
 title: Supported regions
 ---
 
-A crypto holder must reside in one of the countries listed below to be able to order and use Immersve cards.
+A crypto holder must reside in one of the supported regions to be able to
```

Comment:
```suggestion
A crypto holder must reside in one of the regions supported by Immersve to be able to
```

Reply:
I disagree with the suggestion, because the current working is kind of a definition of "supported region". Breaking that apart into "regions supported by Immersve" doesn't feel right to me

Reply:
For me it looked like we had "Supported regions" as the title, and I expect the definition to follow but the current wording just reuses the exact same phrase without expanding on the meaning.
But since everyone else is happy with it I won't die on this hill :)

Reply:
thanks!